app/models/bookmark.rb

```diff
@@ -1,19 +1,80 @@
 # frozen_string_literal: true
 
 class Bookmark < ActiveRecord::Base
+  SPECIAL_BOOKMARKABLE_TYPES = ["Post", "Topic"]
```

Comment:
Sorry for the nit here but special is alittle vague IMO. Perhaps just `DEFAULT_BOOKMARKABLE_TYPES`?

Reply:
Yep no problem!